package.json

```diff
@@ -57,6 +57,10 @@
                       "description": "Specifies whether to perform case-sensitive match.",
                       "type": "boolean"
                     },
+                    "unicode": {
```

Comment:
Is it that the only situation that will be impacted with enabling of unicode option is when someone is already having a regex pattern like `\u{1234}` so that it will stop matching strings like `"u1"` or `"u2"`? Let me know if you can think of more breaks. If not, let's not have a new option and make every regex match unicode by default. The chance of having a user with an extra backslash in the pattern is virtually zero.